src/main/java/PageObjects/WishListsPage.java

```diff
@@ -0,0 +1,99 @@
+package PageObjects;
+import DriverTools.SingleDriver;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Random;
+
+public class WishListsPage {
+    private final WebDriver driver;
+    private final By saveButton = By.id("submitWishlist");
+    private final By wishListsTable = By.id("block-history");
+    private final By products = By.xpath("//a[@class=\"product-name\"]");
+    private final By wishListLink = By.xpath("//tr[contains(@id, 'wishlist_')]/td[1]/a");
+    private final By productTitle = By.id("s_title");
+    private final By deleteListLink = By.xpath("//td[@class='wishlist_delete']/a");
+    private final By wishListNameField = By.id("name");
+    private final By openCartLink = By.xpath("//a[@title='View my shopping cart']");
+    private List<WebElement> productsList;
+
+
+    public WishListsPage() throws IOException {
+        driver = SingleDriver.getSingleDriverInstance().getDriver();
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(saveButton));
+    }
+
+
+    public boolean getWishListsTable() throws NoSuchElementException {
+            boolean result;
+            try {
+                driver.findElement(wishListsTable);
+                result = true;
+            }
+            catch (NoSuchElementException e) {
+                result = false;
+            }
+            return result;
+    }
```

Comment:
1. Give please context, what this method is doing
2. Method getTable returns boolean.. very strange